Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apk --update add \
       su-exec \
       tini && \
     gem update --system && \
-    gem update bundler && \
+    gem install bundler:1.17.2 && \
```

Comment:
I was getting errors about bundler not being installed when building locally.

Reply:
Noted.